src/main/java/org/bitsofinfo/hazelcast/discovery/docker/swarm/DockerSwarmDiscoveryStrategyFactory.java

```diff
@@ -13,15 +13,13 @@
 public class DockerSwarmDiscoveryStrategyFactory implements DiscoveryStrategyFactory {
 
     private static final Collection<PropertyDefinition> PROPERTIES =
```

Comment:
You missed adding `STRICT_DOCKER_SERVICE_NAME_COMPARISON` here

Reply:
Added that now.